frontend/models/product/components/product-option.ts

```diff
@@ -15,7 +15,13 @@ export function productOptionFromShopify(
    variants: ProductVariant[],
    iFixitOptions?: string[]
 ): ProductOption {
-   const values = iFixitOptions?.length ? iFixitOptions : option.values;
+   // TODO: cleanup
+   const values = iFixitOptions?.length
+      ? iFixitOptions.map((option) =>
+           option === 'New-TEsstss' ? 'New' : option
+        )
+      : option.values;
```

Comment:
huh?

Reply:
This hack was necessary because of this: https://github.com/iFixit/react-commerce/issues/1928
Since some time, the Shopify option values are also filtered by iFixit options provided by Akeneo. However Akeneo was returning some malformed data that was causing a faulty display of the variant selector.
I left the `Todo` and deploy_blocked this PR to remember to clean this up.

Since it is now resolved I'm going to remove it immediately